persistence-test/pom.xml

```diff
@@ -52,9 +52,12 @@
       <scope>compile</scope>
     </dependency>
     <dependency>
+      <!-- 09-Sep-2021, tatu: Cassandra 3.11/4.0 backends depend on Jackson 2.9;
```

Comment:
Could probably have worked around this by forcing `jackson-core` and `jackson-databind` versions to 2.12.5, but due to compatibility concerns with backends decided to limit scope here, only boosting to 2.9.10 that C*/DSE use.